### ISSUE TITLE
Engine Tests crash in Mono Latest

### ISSUE THREAD

**Issue by rprouse** [nunit/nunit-console#211]
See https://travis-ci.org/nunit/nunit-console/jobs/217230850

Not sure if we made the Travis Linux/Mono/Latest build no longer required because of this or another failure, but I am also getting this on my local box with Mono 4.8.1 so it isn't a Travis issue.

Most of the error is probably useless to us, but it begins with,

```
* Assertion: should not be reached at sgen-scan-object.h:90
```

and ends with

```
=================================================================
Got a SIGABRT while executing native code. This usually indicates
a fatal error in the mono runtime or one of the native libraries 
used by your application.
=================================================================
```

Not sure what we are going to do about it.

**Comment by rprouse** [comment]
Just did a little searching through the history. This has been happening since #45 where mono:latest was added. It was failing with a similar issue then with Mono 4.2.4.

**Comment by tom-dudley** [comment]
https://github.com/nunit/nunit-console/issues/44 looks to be the same issue.

**Comment by ChrisMaddock** [comment]
This seems to be fixed with the current mono latest, 5.0.1.1. 
https://travis-ci.org/nunit/nunit-console/builds/236200004?utm_source=github_status&utm_medium=notification

mono bug? I'll close, please reopen if people think we should look into it further.

**Comment by rprouse** [comment]
I think it is a mono bug. The nunit framework tests started doing the same thing with Mono 5.0. I wonder if it is fixed in 5.0.1.1 too?
